packages/lodestar-cli/src/cmds/account/cmds/wallet/utils.ts

```diff
@@ -0,0 +1,46 @@
+import fs from "fs";
+import path from "path";
+import {IGlobalArgs} from "../../../../options";
+import {YargsError, writeFile600Perm, randomPassword, readPassphraseFile, initBLS} from "../../../../util";
+import {WalletManager} from "../../../../wallet";
+import {getAccountPaths} from "../../paths";
+import {IWalletRecoverArgs} from "./recover";
+
+export async function createWalletFromArgsAndMnemonic(
+  args: Pick<IWalletRecoverArgs & IGlobalArgs, "name" | "type" | "passphraseFile" | "mnemonicOutputPath" | "rootDir">,
+  mnemonic: string
+): Promise<{uuid: string; password: string}> {
+  await initBLS();
```

Comment:
To be consistent with other commands, could you place this only at the top of each command's handler function?